framework/src/wifi_manager/wifi_manager.c

```diff
@@ -974,3 +982,40 @@ wifi_manager_result_e wifi_manager_scan_ap(void)
 	wifi_manager_result_e res = _handle_request(&msg);
 	return res;
 }
+
+
+wifi_manager_result_e wifi_manager_save_config(wifi_manager_ap_config_s *config)
+{
+	LOCK_WIFIMGR;
+	wifi_utils_result_e res = wifi_profile_write(config);
+	if (res != WIFI_UTILS_SUCCESS) {
```

Comment:
This option is also possible: just make 'wifi_manager_save_config' and 'wifi_manager_remove_config' apis into internal ones. That is, wifi-manager just maintains wifi profiles inside automatically even without any requests.

Reply:
It only support single profile.
If we maintain the wifi profile inside, then a wifi profile is always set even though user doesn't want.
And maintaining inside requires file operation. so I think we should decide the policy very carefully

Reply:
Let me know when you make sure that only one profile is handled without any bugs.

Reply:
I worried that if smaller data is insert then the previous data would be remain at the end of the file. But it's not true. fopen will erase previous data. So we don't need to care about it.
------------------------------------------------------------------------------------------------------
w      Truncate file to zero length or create text file for writing.  The stream is positioned at the beginning of the file.
w+     Open for reading and writing.  The file is created if it does not exist, otherwise it is truncated.  The stream is positioned at the beginning of the file.
------------------------------------------------------------------------------------------------------